src/erlcloud_aws.erl

```diff
@@ -664,6 +669,21 @@ get_metadata_credentials(Config) ->
             get_credentials_from_metadata(Config)
     end.
 
+-spec get_task_metadata_credentials(aws_config()) -> {ok, #metadata_credentials{}} | {error, term()}.
+get_task_metadata_credentials(Config) ->
```

Comment:
one last c/p here to remove and good to go
`get_task_metadata_credentials/1` is not that different for `get_metadata_credentials/1`.
it only differs in calling fun either `get_credentials_from_task_metadata/1` or `get_credentials_from_metadata/1`